src/app/tabs/newAdmin/textrecords/components/SearchTextRecordComponent.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import propTypes from 'prop-types';
+import { multilanguage } from 'redux-multilanguage';
+import {
+  Button, Col,
+} from 'react-bootstrap';
+
+const SearchTextRecordComponent = ({
+  handleClick, strings,
+}) => {
+  const [state, setState] = useState({
+    key: '',
+  });
+
+  return (
+    <div className="major-section records">
+      <h3>{strings.search_text_records}</h3>
+      <div className="row addressInput minor-section contract-abi-view">
+        <div className="row textRecordview">
+          <Col className="col-md-12 label">
+            <p>Resolve txt record key if text record has been previously set:</p>
+            <p>
+              *Custom kets will be displayed in the custom keys table.
+              Global keys allready displayd in the EIP keys table
+            </p>
```

Comment:
These texts should be moved to the strings file so we can have it translated to the other languages.

Reply:
Change addressed. Stings moved to the en file.